### PR TITLE
Use Peril.colorCode everywhere, warn about missing colors (should not happen in production, we would be able to check the logs now)

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

We've got colorCode values from PCMS now, use them instead of fallback

Added logs to track and missing values once we deploy this

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2196** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2196" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2196?utm_source=stack-comment).

File: apps/store/src/components/Perils/Perils.tsx [hunk 1]
```diff
@@ -132,7 +134,16 @@ const Color = styled.div<{ color?: string }>(({ color }) => ({
...
 
+const perilsWithoutColorCode = new Set()
 const titleToColor = (title: string) => {
+  if (!perilsWithoutColorCode.has(title)) {
+    perilsWithoutColorCode.add(title)
+    if (isBrowser()) {
+      datadogLogs.logger.warn(`Peril has no colorCode`, { perilTitle: title })
+    } else {
+      console.warn(`Peril has no colorCode`, { perilTitle: title })
+    }
```
Comment (robinandeer) [inline apps/store/src/components/Perils/Perils.tsx:145]: Maybe we need a utility for this in the future -- think that would be nice!

**Review by robinandeer — APPROVED** [review]
Nice work! I don't see any usage of the `colorCode` property in the PR tho -- but maybe I'm blind 😅

**Comment by alebedev** [comment]
We had `colorCode` usage already in place, all this does it actually start fetching it with product data

**Pull request merged**